### DISCUSSION TITLE
ONNX-Tool Future

### DISCUSSION THREAD

**Discussion by saurabhtangri** [ThanatosShinji/onnx-tool#85] [Q&A]
I work with the ONNX leadership and was wondering if the maintainers would be interested in making this tool part of ONNX?

**Comment by ThanatosShinji** [comment]
Sure, this repo is built for ONNX models. It would be a great pleasure if it could be part of ONNX. Thanks for your attention!

**Reply by saurabhtangri** [reply]
You can see how recently we migrated "Neural compressor" here:  https://github.com/onnx/neural-compressor/tree/main

**Reply by ThanatosShinji** [reply]
I don't have such experience. I suppose that it should be driven by the ONNX owners.
